src/components/dataset/new.vue

```diff
@@ -130,8 +132,19 @@ const hideOrComplete = () => {
 
 <style lang="scss">
 @import '../../assets/scss/variables';
+@import '../../assets/scss/mixins';
 
 #dataset-new-success {
+  display: flex;
+  align-items: center;
+  margin-bottom: 7px;
+
+  > div {
+    @include text-block;
+    width: 80%;
+    margin-left: 3px;
```

Comment:
It looks like the `.icon-check-circle` already has `margin-right`. Could the two margins be combined in a single property?